locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -2782,7 +2814,10 @@ public Boolean call() {
             waitForReachable(checker, connectionDetails, credentialsToTry, setup, timeout);
         } finally {
             for (SshMachineLocation machine : machinesToTry.keySet()) {
-                getManagementContext().getLocationManager().unmanage(machine);
+                if (getManagementContext().getLocationManager().isManaged(machine)) {
+                    // get benign but unpleasant warnings if we unmanage something already unmanaged
+                    getManagementContext().getLocationManager().unmanage(machine);
```

Comment:
This seems like something the location manager should take care of. Either way I assume the equivalent logic should be applied to `waitForWinRmAvailable`.